libs/ignitor.js

```diff
@@ -0,0 +1,10 @@
+
+const ignitor = (original) => (evt, ctx, cb) => {
+  if (evt.ignitor) {
+    return cb(null, 'pinged');
+  }
+
+  return original(evt, ctx, cb);
+};
+
+module.exports = { ignitor };
```

Comment:
default wrapper is now a fully function module, and will be injected as part of the build process.